### PR TITLE
Reworked start page

### PR DESCRIPTION
**Pull request by paulblakehmrc**
### Description of Work

New copy on start page.
Removed section from Title component - it's not used anywhere

#### Check list 
 - [x] `./precheck` was executed (Integration/Component/Unit tests)
 - [x] `sbt scalafmt test:scalafmt` was executed
 - [ ] Required Environment Config has been amended/added
 - [ ] Links to dependencies have been included (BE/FE work)
 - [x] User Acceptance Tests (UAT) were run locally and have passed.
 - [x] No Accessibility errors/warnings reported by Wave

### CODE REVIEW

File: app/uk/gov/hmrc/plasticpackagingtax/registration/views/model/Title.scala [hunk 1]
```diff
@@ -20,22 +20,14 @@ import play.api.i18n.Messages
 
 case class Title(
   headingKey: String,
-  sectionKey: String = "",
   headingArg: String = "",
```
Comment (SteveSugden) [inline app/uk/gov/hmrc/plasticpackagingtax/registration/views/model/Title.scala:23]: Are we removing section names form all titles?

File: app/uk/gov/hmrc/plasticpackagingtax/registration/views/start_page.scala.html [hunk 2]
```diff
@@ -31,66 +32,95 @@
...
+
+@govukLayout(title = Title("startPage.title")) {
+
+<div>
+    @pageHeading(messages("startPage.title"))
+    @paragraphBody(message = messages("startPage.description"), classes = "govuk-body", id = Some("description"))
+</div>
+
+<div>
+    <h2 id="when-to-register" class="@gdsPageHeading">@messages("startPage.whenToRegister")</h2>
```
Comment (RyanKellyHMRC) [inline app/uk/gov/hmrc/plasticpackagingtax/registration/views/start_page.scala.html:102]: Can we use the `subHeading` component which was introduced recently?

File: app/uk/gov/hmrc/plasticpackagingtax/registration/views/start_page.scala.html [hunk 3]
```diff
@@ -31,66 +32,95 @@
...
     )
+}
+
+
+@govukLayout(title = Title("startPage.title")) {
+
+<div>
+    @pageHeading(messages("startPage.title"))
+    @paragraphBody(message = messages("startPage.description"), classes = "govuk-body", id = Some("description"))
+</div>
```
Comment (RyanKellyHMRC) [inline app/uk/gov/hmrc/plasticpackagingtax/registration/views/start_page.scala.html:99]: Do these `<div>`s serve any purpose?

File: conf/messages.en [hunk 4]
```diff
@@ -1,10 +1,9 @@
 feedback=This is a new service - your {0} will help us to improve it.
 feedback.link=feedback
 
-service.name=Plastic Packaging Tax
+service.name=Register for Plastic Packaging Tax
```
Comment (RyanKellyHMRC) [inline conf/messages.en:4]: Are there any knock-on effects of changing this?

Comment (paulblakehmrc) [reply conf/messages.en:4]: Haven't come across any but can talk about this if you like.

Comment (RyanKellyHMRC) [reply conf/messages.en:4]: Sounds like @SteveSugden is also convinced that this will not cause a problem.

**Pull request merged**